nengo_viz/static/viz_hotkeys.js

```diff
@@ -0,0 +1,23 @@
+VIZ.Hotkeys = function () { 
+
+    document.addEventListener('keydown', function(ev) {
+        // undoo with ctrl-z
+        if (ev.ctrlKey == true && ev.keyCode == 90) {
+            VIZ.netgraph.notify({ undo: "1" });
+        }
+        // redo with shift-ctrl-z
+        if (ev.ctrlKey == true && ev.shiftKey == true && ev.keyCode == 90) {
+            VIZ.netgraph.notify({ undo: "0" });
+        }
+        // redo with ctrl-y
+        if (ev.ctrlKey == true && ev.keyCode == 89) {
+            VIZ.netgraph.notify({ undo: "0" });
+        }
+        // run model with spacebar
+        if (ev.keyCode == 32) {
+            sim.on_pause_click();
+        }
+    });
+}
+
+VIZ.Hotkeys = new VIZ.Hotkeys();
```

Comment:
This should be `VIZ.hotkeys`? Or something to make it different from the `VIZ.Hotkeys` function, which is no longer accessible after this line (since you're overwriting the `VIZ.Hotkeys` function with the results of this call).